src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -1202,6 +1202,11 @@ def _get_authorization_code_worker(authority_url, resource, results):
             break
         except socket.error as ex:
             logger.warning("Port '%s' is taken with error '%s'. Trying with the next one", port, ex)
+        except UnicodeDecodeError:
```

Comment:
>except UnicodeDecodeError: [](start = 8, length = 26)

make sure we handle this exception in MSAL too

Reply:
Sure.